audio_to_multiple_pose_gan/my_audio_to_pose_net/my_first_model.py

```diff
@@ -242,21 +242,33 @@ def preprocess_dataset(files):
 # compile, aka define the optimizer and loss
 model.compile(
     optimizer=tf.keras.optimizers.Adam(),
-    loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True),
+    loss=tf.keras.losses.CategoricalCrossentropy(from_logits=True),
```

Comment:
SparseCategoricalCrossentropy vs CategoricalCrossentropy just indicated if you expect the data to be one-hot encoded